node/src/main/resources/defaults.conf

```diff
@@ -317,6 +317,12 @@ casper {
     # Allow node being ceremony master
     # If this is `true` and no genesis block available on startup node will start genesis ceremony.
     ceremony-master-mode = ${standalone}
+
+    # Allow node starts from genesis block or Approved block
+    # If this is `true`,when a node without any data connecting to exist network, the node would starts from
+    # genesis block and validate blocks from scratch. Otherwise, the node would start from approved block which
+    # can save a lot of time of validating all the history blocks
+    from-last-finalized-state = false
```

Comment:
@zsluedem I believe better to have `trimState` option as I named it first place, and have it true by default.
Most of the time we need just connect to the network not thinking about what os needed for that.
And only in rare cases we need to go back to the very first block (which actually can be not genesis on the particular node)

Reply:
Yeah. I agree with you. But I am not sure if last-finalized-state is mature enough. I want to make this default to be `true`  when we think it is really ready.

Reply:
We are working on feature branch for LFS so we can assume it will be ready when merging to dev.

Reply:
ok.